frame/vamm/src/mock.rs

```diff
@@ -0,0 +1,125 @@
+// use super::currency_factory::MockCurrencyId;
+// use frame_support::{
+// 	traits::GenesisBuild,
+// 	PalletId,
+// };
+// use num_traits::Zero;
+// use orml_traits::parameter_type_with_key;
+// use sp_runtime::{
+// 	traits::IdentityLookup,
+// };
+
+use crate as pallet_vamm;
+
+use frame_support::{
+	parameter_types,
+	traits::Everything,
+};
+
+use sp_runtime::{
+	testing::Header,
+	traits::IdentityLookup
+};
+use sp_core::H256;
+
+pub type BlockNumber = u64;
+pub type AccountId = u128;
+pub type Balance = u128;
+
+pub const ALICE: AccountId = 0;
+
+// ----------------------------------------------------------------------------------------------------
+//                                                Config                                               
```

Comment:
Nitpick: since you commented on other config sections with the name of the pallet, wouldn't it make sense to do the same here? E.g., 'FRAME system'. 

I think I'll do this on mine as well

Reply:
Yeah, seems better this way. 
Thanks!